clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -98,27 +133,46 @@ export const useHasPlus = () => {
 };
 
 const emptyClassifyInstances: ClassifyInstanceResponseValues[] = [];
-export const selectClassifyInstances = createSelector(
-  plusApi.endpoints.getAllClassifyInstances.select(),
+export const selectDatasetClassifyInstances = createSelector(
+  plusApi.endpoints.getAllClassifyInstances.select({
+    resource_type: GenerateTypes.DATASETS,
+  }),
   ({ data: instances }) => instances ?? emptyClassifyInstances
 );
 
+export const selectSystemClassifyInstances = createSelector(
+  [(state) => state, selectSystemsForReview],
+  (state, systems) =>
+    plusApi.endpoints.getAllClassifyInstances.select({
+      resource_type: GenerateTypes.SYSTEMS,
+      fides_keys: systems.map((s) => s.fides_key),
+    })(state)?.data ?? emptyClassifyInstances
+);
+
 const emptyClassifyInstanceMap: Map<string, ClassifyInstanceResponseValues> =
   new Map();
-export const selectClassifyInstanceMap = createSelector(
-  selectClassifyInstances,
-  (instances) => {
-    const map = new Map<string, ClassifyInstanceResponseValues>();
-    instances.forEach((ci) => {
-      map.set(ci.dataset_key, ci);
-    });
-
-    if (map.size === 0) {
-      return emptyClassifyInstanceMap;
-    }
-
-    return map;
+
+const instancesToMap = (instances: ClassifyInstanceResponseValues[]) => {
+  const map = new Map<string, ClassifyInstanceResponseValues>();
+  instances.forEach((ci) => {
+    map.set(ci.dataset_key, ci);
```

Comment:
This code uses the `dataset_key` key, so it _shouldn't_ work for the systems map which [looks up](https://github.com/ethyca/fides/blob/0da90bcb8efba4ac10cf181b757481bbd5c816a3/clients/admin-ui/src/features/system/ClassifySystemsTable.tsx#L24) by `system.fides_key`. I hope this API isn't putting the system key on the dataset key field 🤞😑🤞

Reply:
it is... 😬 @SteveDMurphy and I talked about changing the key to `fides_key` instead but I don't think we ever made an issue for it. mostly didn't want to break the existing dataset classification (the original API was written in a way that never expected to do anything other than datasets, but here we are 😄 ) 

I just made a ticket https://github.com/ethyca/fidesctl-plus/issues/326